src/resources/theme/main.globals.ts

```diff
@@ -28,13 +28,16 @@ export const mainStyles = css`
     --margin-title-rtl: 0 24px 0 0;
 
     /* safe-area-insets */
-    --safe-area-inset-top: var(--app-safe-area-inset-top, env(safe-area-inset-top, 0));
-    --safe-area-inset-bottom: var(--app-safe-area-inset-bottom, env(safe-area-inset-bottom, 0));
-    --safe-area-inset-left: var(--app-safe-area-inset-left, env(safe-area-inset-left, 0));
-    --safe-area-inset-right: var(--app-safe-area-inset-right, env(safe-area-inset-right, 0));
+    --safe-area-inset-top: max(var(--app-safe-area-inset-top, env(safe-area-inset-top, 0)), 0px);
+    --safe-area-inset-bottom: max(var(--app-safe-area-inset-bottom, env(safe-area-inset-bottom, 0)), 0px);
+    --safe-area-inset-left: max(var(--app-safe-area-inset-left, env(safe-area-inset-left, 0)), 0px);
+    --safe-area-inset-right: max(var(--app-safe-area-inset-right, env(safe-area-inset-right, 0)), 0px);
 
     --safe-area-inset-y: calc(var(--safe-area-inset-top, 0px) + var(--safe-area-inset-bottom, 0px));
     --safe-area-inset-x: calc(var(--safe-area-inset-left, 0px) + var(--safe-area-inset-right, 0px));
+
+    --safe-width: calc(100vw - var(--safe-area-inset-left, 0px) - var(--safe-area-inset-right, 0px));
```

Comment:
We already fallback to zero above right?

```suggestion
    --safe-width: calc(100vw - var(--safe-area-inset-left) - var(--safe-area-inset-right));
```